lua/oil/util.lua

```diff
@@ -697,4 +697,54 @@ M.adapter_list_all = function(adapter, url, opts, callback)
   end)
 end
 
+M.send_to_quickfix = function(opts)
```

Comment:
Could you add a type annotation for `opts`? It helps document what the available options are, and provides some type safety. You may also want to include some default values, or make the type non-nilable.

Reply:
It now defaults to replacing items in the quickfix list.